src/UserSettingsPage.cpp

```diff
@@ -477,9 +478,9 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
 
         QFontDatabase fontDb;
         auto fontFamilies = fontDb.families();
-        for (const auto &family : fontFamilies) {
-                fontSelectionCombo_->addItem(family);
-        }
+        // for (const auto &family : fontFamilies) {
+        //         fontSelectionCombo_->addItem(family);
+        // }
```

Comment:
I feel it's best to remove the lines of code that are not needed instead of commenting out?